Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -161,7 +161,18 @@ static bool restore_string_position(MatchInput const& input, MatchState& state)
 OwnPtr<OpCode> ByteCode::s_opcodes[(size_t)OpCodeId::Last + 1];
 bool ByteCode::s_opcodes_initialized { false };
 size_t ByteCode::s_next_checkpoint_serial_id { 0 };
-u32 StringTable::next_serial { 0 };
+static u32 s_next_string_table_serial { 0 };
+
+StringTable::StringTable()
```

Comment:
@alimpfard now this patch has an actual change to LibRegex: moving this ctor/dtor out of line. I don't expect it to actually affect anything important though.

Reply:
sorry, I forgot to reply to this. that change looked fine and continues to look fine :)